CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_marketo_source v0.8.0
```

Comment:
Reminder to add whatever relevant updates from this PR review into the CHANGELOG.

Reply:
Done.